vhdbuilder/packer/install-dependencies.sh

```diff
@@ -132,6 +132,7 @@ capture_benchmark "${SCRIPT_NAME}_validate_container_runtime_and_override_ubuntu
 
 # Configure SSH service during VHD build for Ubuntu 22.10+
```

Comment:
should we add the check in this file instead, to call `configureSSHService` only if on the right OS.  I think the majority of the helper functions simply execute the required logic, the Decisions is all done in this file for other logic.

Reply:
I agree with this, I think it's implement the way it is currently since we were originally thinking it would be part of CSE, but now that we're doing it during the build I think adding the OS version check out here makes more sense

Reply:
Oh right. This makes sense to me too. Didn't realize this in the previous review.